### ISSUE TITLE
Some prompts in RDS fields not readable for screenreader

### ISSUE THREAD

**Issue by NightPurrer** [TomaszGasior/RadioLista-v3#154]
When i filling rds fields, some prompts are read and others are not.
Maybe some of the explanations not exist, or maybe they are'nt readable for the NVDA screenreader.
Unfortunately, I cannot check what version is true.
Here's the list with my comments:
Programme Identification: Prompt not exist or not readable.
Programme Service: Prompt is readable: "na przykład: POLSKIE|RADIO|JEDYNKA"
Radio Text: Prompt readable: "każdy komunikat w osobnej linii"
Programme Type: Prompt not exist or not readable.

**Comment by TomaszGasior** [comment]
It seems to be you misunderstood the meaning of the prompt itself in the form but I will check the details in my virtual machine with Windows and NVDA and give you more information, as soon as I will find free time.

**Comment by TomaszGasior** [comment]
Thank you for your bug report. I tried to reproduce your problem with Windows 10, Chrome and NVDA 2021.1. Indeed, I was able to reproduce the behavior you describe. However, this behavior is correct. 

The "RDS" form has four fields. All of them have basic label for end users and for screen readers. Second field called "Programme Service" has some extra information with example how to use it. Third field called "Radio Text" also has extra information about how to properly fill it in. First and fourth fields — "Programme Identification" and "Programme Type" are simple enough to not have any extra descriptions. :)
